vda5050_tb3_adapter/vda5050_tb3_adapter/tb3_adapter.py

```diff
@@ -239,7 +239,7 @@ def process_vda_action_callback(self, goal_handle):
             ]:
                 feedback_msg.current_action.action_status = action_status
                 goal_handle.publish_feedback(feedback_msg)
-                time.sleep(5)
+                time.sleep(3)
```

Comment:
Is there a reason for this update? Maybe we can leave it as a comment?

Reply:
I just wanted to shorten a little bit the time it takes for this dummy action to execute.